shell/platform/windows/platform_handler.cc

```diff
@@ -227,6 +227,7 @@ PlatformHandler::PlatformHandler(
       return std::make_unique<ScopedClipboard>();
     };
   }
+  FlutterDesktopPluginRegistrarRegisterTopLevelWindowProcDelegate(engine_->GetRegistrar(), PlatformHandler::WindowProcCallback, this);
```

Comment:
Instead of using the plugin API, could we instead use `WindowProcDelegateManager::RegisterTopLevelWindowProcDelegate` directly?